frontend/src/pages/ProcessesPage.tsx

```diff
@@ -2,19 +2,20 @@ import { Button } from "../components/Button";
 import { Table } from "../components/Table";
 import { useUIStore } from "../contexts/useUIStore";
 import { gql, useQuery } from "@apollo/client";
+import { NavLink } from "react-router-dom";
 
 const PROCESSES = gql`
   query Test {
```

Comment:
This query should have a better name since it is used now for more than testing.